fiftyone/core/collections.py

```
@@ -103,7 +103,72 @@ def submit(self, fn, *args, **kwargs):
+    asynchronous batch writing.
+
+    Subclasses must override :meth:`_do_save_batch` to perform the actual
+    write operation.
+
+    Args:
+        max_workers (1): the number of worker threads
+        async_writes (True): whether to use a real executor or the synchronous
+            ``_DummyExecutor``
+    """
+
+    def __init__(self, max_workers=1, async_writes=True):
```

Ruff (0.15.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)